backend/controllers/inviteContacts.ts

```diff
@@ -35,4 +36,57 @@ export default class InviteContactController {
       })
     }
   }
+
+  static async verifyPlanInvite(req: any, res: Response, next: NextFunction): Promise<void> {
+    try {
+      const planInviteID = req.params.id
+
+      // if no cookie, redirect to log in
+      if (req.user === null || req.user === undefined) {
+        res.status(200).json({
+          reason: 'NOT_LOGGED_IN',
+          planInviteID
+        })
+        return
+      }
```

Comment:
Hmm, should this be done here, or is this not handled already in your route middleware?

also, if we're going to return a status code here, it would be 401